.github/workflows/test.yml

```diff
@@ -18,3 +18,34 @@ jobs:
         with:
           route: GET /repos/:owner/:repo/releases/latest
       - run: "echo latest release: '${{ steps.get_latest_release.outputs.data }}'"
+      # Create check run
+      - uses: ./
+        id: create_check_run
+        with:
+          route: POST /repos/:owner/:repo/check-runs
+          headers: '{"accept": "application/vnd.github.antiope-preview+json"}'
```

Comment:
You can do 

```suggestion
          mediaType: '{"previews": ["antiope"]}'
```

Reply:
It looks like `mediaType` doesn't currently work. Could it be because it is `toLowerCase()`'d [here](https://github.com/octokit/request-action/blob/master/index.js#L59)?

It comes through in the request body like:
```
"mediatype":{"previews":["antiope"]}
```

Reply:
Hmmm yeah that could be, we will need to implement an exception for mediaType I think

Reply:
Cool, see what you think about https://github.com/octokit/request-action/pull/3, then I'll add these `mediaType` changes here. 🙇